app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -31,12 +39,9 @@ const Note = ({ note }) => (
       </div>
       <div className="mt-3 flex items-center space-x-1">
         <Clock size={15} />
-        <Tooltip
-          content={dayjs(note.createdAt).format("dddd, hh:mmA")}
-          position="bottom"
-        >
+        <Tooltip content={formatTime(note.createdAt)} position="bottom">
           <Typography style="body3">
-            Created {dayjs(note.createdAt).fromNow()}{" "}
+            Created {createdAt(note.createdAt)}
```

Comment:
`createdAt` is a vague util name. If I am reading it then what I understand is that it returns the `createdAt` value. But here you might be actually formatting any particular date into the format that aligns with requirements of displaying the `createdAt` value in UI.

So please find a better name for this util function.